### PR TITLE
Add home page

### PR DESCRIPTION
**Pull request by ryckutis**

### CODE REVIEW

File: CarLease-Co/src/app/app.routes.ts [hunk 1]
```diff
@@ -12,11 +11,12 @@ import { LeaseApplicationsListComponent } from './leases/lease-applications-list
...
 import { LoginFormComponent } from './login/login-form/login-form.component';
 import { SysAdminViewComponent } from './sys-admin-view/sys-admin-view/sys-admin-view.component';
+import { HomePageComponent } from './home-page/home-page.component';
 
 export const routes: Routes = [
   {
     path: ROUTES.HOME,
-    component: AppComponent,
+    component: HomePageComponent,
     canActivate: [loginAccessGuard],
```
Comment (WlatonSimons) [inline CarLease-Co/src/app/app.routes.ts:20]: Is this the no login landing page? The guard will route you to the login page all the time. If not then carry on.

File: CarLease-Co/src/app/home-page/home-page.component.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<main>
+  <div class="content">
+    <div class="left">
```
Comment (WlatonSimons) [inline CarLease-Co/src/app/home-page/home-page.component.html:3]: Usually you'd want to follow some naming convention for your classes eg: https://getbem.com/naming/

.form { }
.form--theme-xmas { }
.form--simple { }
.form__input { }
.form__submit { }
.form__submit--disabled { }

but tis just an fyi

File: CarLease-Co/src/app/home-page/home-page.component.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      <h1>Cheapest and easiest way to lease a car on the planet earth</h1>
+      <p>Try CarLess and see how much fun car leasing can be.</p>
+      <button (click)="redirectToLeaseApplicationForm()">Get Started</button>
+    </div>
+    <div class="right">
+      <img src="../../assets/porsche.png" alt="Porsche" />
```
Comment (WlatonSimons) [inline CarLease-Co/src/app/home-page/home-page.component.html:9]: Usually best to store links/paths in a variable and pass it.

File: CarLease-Co/src/app/app.routes.ts (hunk 1 above)
Comment (ryckutis) [reply CarLease-Co/src/app/app.routes.ts:20]: This home component displayed only after successful login

**Review by WlatonSimons — APPROVED** [review]

File: CarLease-Co/src/app/home-page/home-page.component.html (hunk 3 above)
Comment (ryckutis) [reply CarLease-Co/src/app/home-page/home-page.component.html:9]: Use routerLink instead of button u mean?

Comment (WlatonSimons) [reply CarLease-Co/src/app/home-page/home-page.component.html:9]: nah just do Porsche to pass the path as a variable.

Comment (WlatonSimons) [reply CarLease-Co/src/app/home-page/home-page.component.html:9]: best to hold links and paths as a constant and then use them in the templates.

**Pull request closed without merging**